linode/manager
# fix: [M3-6188] - Fix TopMenuIcon tooltips

## Description 📝
Fixes `TopMenu` issues via the `tab` key navigation which resulted in the `ToolTip` not being shown in the following icons: `TopMenuIcon`, `Help`, and `Community`.

## Major Changes 🔄
- Corrects the function name from `Help` to `Community` in the `Community.tsx` file.
- Replaces default exports with named exports.
- Removes usage of the `Link` component and replaces it with the `StyledLinkButton` component.
- Adds verbiage to the `Community` icon tooltip to inform the user that the link will open in a new tab.

## Preview 📷

| Before  | After   |
| ------- | ------- |
| ![Img-00](https://github.com/linode/manager/assets/119514965/0b8f3f5f-7b25-4bd1-96fc-dee686148e91) | ![Img-02](https://github.com/linode/manager/assets/119514965/958c5ed7-779d-484b-ad40-e98b48c1410c) |
| ![Img-01](https://github.com/linode/manager/assets/119514965/87cda2c0-a2ba-4480-a377-f586037538bb) | ![Img-03](https://github.com/linode/manager/assets/119514965/705a1728-7783-427f-8268-a19afe4a1117) |


## How to test 🧪
1. Navigate the `TopMenu` via the `tab` key and inspect that the appropriate tooltip is shown upon focus.
2. Hover over the`TopMenu` icons and inspect that the appropriate tooltips are shown upon accordingly.
3. Click or press the `enter`/`return` key on the `Help` and `Community` icons to test that each link works as intended
